luci-app-overthebox/luasrc/view/overthebox/wanstatus.htm

```diff
@@ -204,6 +204,18 @@
 					var equipmentIcon = String.format('<embed id="modem_%s" onload="setColorSVG(\'modem_%s\', \'%s\')" src="<%=resource%>/modem.svg" />', mArray.wans[i].name, mArray.wans[i].name, mArray.wans[i].color);
 					var title = mArray.wans[i].label + " (" + mArray.wans[i].gateway + ")";
 					var content = String.format('%s<br />wan address: <strong>%s</strong><br />whois: %s<br />ping: %s ms<br />Multipath: %s', stat, wanip, whois ? whois : "", curping, multipath);
+
+					if(mArray.wans[i].shaping != "off")
+					{
+						var upload = "off";
+						var download = "off";
+						if(mArray.wans[i].upload_state == "up")
```

Comment:
Maybe you should add delimiter for the if block.
Remember [Apple GOTO FAIL](https://nakedsecurity.sophos.com/2014/02/24/anatomy-of-a-goto-fail-apples-ssl-bug-explained-plus-an-unofficial-patch/)